### PR TITLE
Add script for checking gem owners

### PR DESCRIPTION
**Pull request by GregSutcliffe**
As per discussion on IRC as to where to home this, I've added it here. Usage is simple enough, add "--official" to the call to only match gems with "theforeman" somewhere in their URIs. Sample output is about to be sent to the dev list for previewing.

### CODE REVIEW

File: scripts/check_plugin_authors.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/ruby
```
Comment (ares) [inline scripts/check_plugin_authors.rb]: `#!/usr/bin/env ruby` perhaps?

File: scripts/check_plugin_authors.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      next if ( ARGV[0] == "--official" and not gem.select { |k,v| k =~ /uri/}.values.join(" ") =~ regex )
+      gems << gem['name']
+    end
+    page += 1
+  end
+  gems.sort
+end
+
+foreman_gems = get_gem_names('foreman')
+proxy_gems   = get_gem_names('smart_proxy')
```
Comment (ares) [inline scripts/check_plugin_authors.rb]: maybe `smart-proxy` too?

Comment (GregSutcliffe) [reply scripts/check_plugin_authors.rb]: I ran that with no hits, but yeah, should add it to the search anyway

File: scripts/check_plugin_authors.rb (hunk 1 above)
Comment (GregSutcliffe) [reply scripts/check_plugin_authors.rb]: Good catch, will fix later, gotta run :P

**Comment by GregSutcliffe** [comment]
@ares - updated, sorry for the delay.

**Comment by GregSutcliffe** [comment]
ping @ares?

File: scripts/check_plugin_authors.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+require 'json'
+require 'rest-client'
+
+def get_gem_names(search)
+  page=1
+  gems=[]
+  url="https://rubygems.org/api/v1/search.json?query=#{search}"
+
+  response = RestClient.get('https://google.com') # test connection and initialize "response"
```
Comment (ekohl) [inline scripts/check_plugin_authors.rb]: Wouldn't it be better to get rubygems.org?

Comment (GregSutcliffe) [reply scripts/check_plugin_authors.rb]: fair enough ;)

**Comment by ares** [comment]
Looks good, thanks @GregSutcliffe and @ekohl, merging.

**Pull request merged**